src/pages/api/pdf.ts

```diff
@@ -0,0 +1,582 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import hummus from 'hummus'
+import { TaxForm } from '../../types/TaxForm'
+
+const FIRST_COLUMN = 31.5
+const BOX_WIDTH = 14.4
+
+class PdfTemplate {
+  pdfWriter: any
+  pageModifier: any
+  fontOptions = {
+    font: null,
+    size: 12,
+    colorspace: 'gray',
+    color: 0x00,
+  }
+  currentPage: number
+  dic: string
+
+  constructor(res: NextApiResponse) {
+    this.pdfWriter = hummus.createWriterToModify(
+      new hummus.PDFRStreamForFile('./src/pdf-assets/template2019.pdf'),
```

Comment:
Tu by to asi chcelo pouzit absolutnu cestu cez `__dirname` a malo by to fungovat na vsetkych platformach